.github/workflows/hugegraph-llm.yml

```
@@ -0,0 +1,199 @@
+        source .venv/bin/activate
+        uv pip install pytest pytest-cov
+        # Debug: Check current directory and file existence
+        echo "=== Debug Information ==="
+        echo "Current working directory: $(pwd)"
+        echo "Listing current directory:"
+        ls -la
+        echo "Checking hugegraph-llm directory:"
+        ls -la hugegraph-llm/
+        echo "Checking dependency files:"
+        ls -la hugegraph-llm/requirements.txt hugegraph-llm/pyproject.toml 2>/dev/null || echo "Some dependency files not found"
+        
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)